graph-commons/src/main/scala/io/renku/triplesstore/model.scala

```diff
@@ -32,7 +32,7 @@ object DatasetName extends TinyTypeFactory[DatasetName](new DatasetName(_)) with
   implicit val configReader: ConfigReader[DatasetName] = stringTinyTypeReader(DatasetName)
 }
 
-trait DatasetConfigFile extends StringTinyType
+trait DatasetConfigFile extends Any with StringTinyType
```

Comment:
Does the `extends Any` make a practical difference (maybe for the refined lib)? Or maybe `extends AnyVal` could be an option (looking at the sub types)?